runtime/failpoint.go

```diff
@@ -16,10 +16,12 @@ package runtime
 
 import (
 	"fmt"
+	"sync"
 )
 
 type Failpoint struct {
-	t *terms
+	t           *terms
+	failpointMu sync.RWMutex
```

Comment:
For me name `failpointMu` clashes with the global `failpointsMu`. Name is local to failpoint so there is no need to prefix it's name.
```suggestion
	mux sync.RWMutex
```

Reply:
+1